deployments/rds-s3/base/kustomization.yaml

```diff
@@ -6,23 +6,23 @@ resources:
   - ../../../upstream/common/cert-manager/cert-manager/base
   - ../../../upstream/common/cert-manager/kubeflow-issuer/base
   # Istio
-  - ../../../upstream/common/istio-1-14/istio-crds/base
-  - ../../../upstream/common/istio-1-14/istio-namespace/base
-  - ../../../upstream/common/istio-1-14/istio-install/base
+  - ../../upstream/common/istio-1-16/istio-crds/base
+  - ../../upstream/common/istio-1-16/istio-namespace/base
+  - ../../upstream/common/istio-1-16/istio-install/base
   # OIDC Authservice
   - ../../../upstream/common/oidc-authservice/base
   # Dex
   - ../../../upstream/common/dex/overlays/istio
   # KNative
   - ../../../upstream/common/knative/knative-serving/overlays/gateways
   - ../../../upstream/common/knative/knative-eventing/base
-  - ../../../upstream/common/istio-1-14/cluster-local-gateway/base
```

Comment:
Wouldn't this go to `kubeflow-manifests/deployments/` instead of `kubeflow-manifests/`

Reply:
good catch. updated.